.claude/skills/java-coding-standards/SKILL.md

```
@@ -0,0 +1,244 @@
+- Avoid raw types; declare generic parameters
+- Prefer bounded generics for reusable utilities
+
+```java
+public <T extends Identifiable> Map<Long, T> indexById(Collection<T> items) { ...}
+```
+
+## Project Structure
+
+### Maven/Gradle
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)